src/dcat-ap/dcat-dataset.ts

```diff
@@ -3,10 +3,14 @@ import { localeToLang } from './dcat-formatters';
 import * as _ from 'lodash';
 import { isPage } from '@esri/hub-sites';
 import { UserSession } from '@esri/arcgis-rest-auth';
+import { IModel, datasetToContent, DatasetResource, getContentSiteUrls, IHubRequestOptions, getProxyUrl, datasetToItem } from '@esri/hub-common';
 
 // Required fields from the API
 export const defaultRequiredFields = [
   'id',
+  'access', // needed for proxied csv's
+  'size', // needed for proxied csv's
```

Comment:
Just as a heads up, the Portal Search API does _NOT_ return size with search results, last time I checked. This _technically_ is a Hub enrichment.